### PR TITLE
Service

### PR DESCRIPTION
**Pull request by bemze**
CSS update (style, btn style responsive)

Idomi situacija: ant Martyno video branco padirbau service.css faile ir padariau commit. Veliau tesiau darba pas save service branche. Atrodo viskas sustailinta, bet bus idomu pamatyti, kaip suveiks mano klaida

### CODE REVIEW

**Comment by bemze** [comment]
conflict fixed

File: css/components/button.css [hunk 1]
```diff
@@ -7,6 +7,7 @@
     color: #fff;
     font-size: 15px;
     height: 54px;
+    line-height: 54px;
```
Comment (pigma15) [inline css/components/button.css:10]: image

siuo atveju line height palieka teksta apacioje, kadangi jis tokio pacio dydzio, kaip mygtuko height

File: css/components/services.css [hunk 2]
```diff
@@ -9,28 +9,73 @@
...
-                border: 1px solid var(--font-blue);
-                top: -1px;
-                right: -1px;
-                bottom: -1px;
-                left: -1px;
-                animation: pulse 2s cubic-bezier(0, 0, 0.5, 1) infinite;
-                opacity: 0;
-            }
\ No newline at end of file
+    left:40%;
```
Comment (pigma15) [inline css/components/services.css]: image
```
left: 50%;
transform: translateX( -50%)
```

**Comment by bemze** [comment]
conflicts solved

**Review by pigma15 — DISMISSED** [review]
Approved for Mantas to be able to work further. Edits are going to be done later

File: css/components/button.css (hunk 1 above)
Comment (bemze) [reply css/components/button.css:10]: Bug was fixed in BTN branch  PR, but it is not approved till now:(

File: css/components/services.css (hunk 2 above)
Comment (bemze) [reply css/components/services.css]: done

**Review by martynas3113 — APPROVED** [review]
Atrodo lyg ir gerai

**Review by pigma15 — APPROVED** [review]
incomming :)

**Pull request merged**